### PR TITLE
Bump CI to 22.04

### PR DESCRIPTION
**Pull request by aleclearmind**

### CODE REVIEW

File: docs/writing_components.md [hunk 1]
```diff
@@ -100,38 +100,3 @@ components:
...
-  caliban:
-    repository: caliban
-    builds: #@ typical_cmake_builds(**build_args())
-```
-
-### Common optimization levels
-
-`.orchestra/config/lib/optimization_flavors.lib.yml` 
-contains some common options useful for defining various builds.
-
```
Comment (depau) [inline docs/writing_components.md]: I think you deleted more than intended

File: .orchestra/config/components/toolchain/msvc.yml [hunk 2]
```diff
@@ -172,21 +172,32 @@ toolchain/(@= name @)/vc:
...
-        trap 'kill "$MSPDBSRV_PID"' EXIT;
+
+        # Stop mspdbsrv.exe upon exit
+        trap 'wine "$SCRIPT_DIR"/(@= bin @)/mspdbsrv.ex*e -stop' EXIT;
+
+        # Wait for mspdbsrv.exe to be up and running
+        while ! [[ "$(readlink -f "/proc/$MSPDBSRV_PID/exe")" == *"wine"* ]]; do
+            sleep 0.2
+        done
+        sleep 0.2
```
Comment (mrjackv) [inline .orchestra/config/components/toolchain/msvc.yml:192]: Is this going to work if the server is (over-)loaded? (e.g. CI running 2 builds?)

File: .orchestra/config/components/toolchain/msvc.yml [hunk 3]
```diff
@@ -172,21 +172,32 @@ toolchain/(@= name @)/vc:
...
         # Suppress wine debug information
-        export WINEDEBUG=-all
+        export WINEDEBUG="-all"
+
+        # Disable .Net and mshtml
+        export WINEDLLOVERRIDES="mscoree,mshtml="
 
         # Run one dedicated mspdbsrv.exe per instance of cl.exe
-        export _MSPDBSRV_ENDPOINT_="$(python -c 'import uuid; print(str(uuid.uuid4()))')"
+        export _MSPDBSRV_ENDPOINT_="$(python3 -c 'import uuid; print(str(uuid.uuid4()))')"
```
Comment (mrjackv) [inline .orchestra/config/components/toolchain/msvc.yml]: You can shorten to `python3 -c 'import uuid; print(uuid.uuid4())'`

File: .orchestra/config/components/toolchain/msvc.yml (hunk 2 above)
Comment (aleclearmind) [reply .orchestra/config/components/toolchain/msvc.yml:192]: As per our discussion, can't do much better than this easily.

**Pull request closed without merging**